gridpath/transmission/operations/operational_types/tx_simple.py

```diff
@@ -158,12 +187,24 @@ def add_model_components(
         )
     )
 
+    m.TX_SIMPLE_BLN_TYPE_HRZS_W_MIN_CONSTRAINT = Set(
+        dimen=3, within=m.TX_SIMPLE * m.BLN_TYPE_HRZS
+    )
+
     # Params
     ###########################################################################
     m.tx_simple_loss_factor = Param(
         m.TX_SIMPLE, within=PercentFraction, default=0
     )
 
+    # Optional Params
+    ###########################################################################
+
+    m.tx_simple_min_transmit_power_mw = Param(
+        m.TX_SIMPLE_BLN_TYPE_HRZS_W_MIN_CONSTRAINT,
+        within=Reals, default=0
```

Comment:
@Janie115, I think it would be better to default to -Infinity here, which would make it possible to skip the `if` logic in the `min_transmit_power_rule()` below. Simply set a variable called Infinity at the top of the module like this` Infinity = float('inf')`. There's an example in gridpath/system/markets/volume.py.

Reply:
@anamileva I made the modification.